CHANGELOG.unreleased.md

```diff
@@ -8,6 +8,7 @@
 > Users must be able to say: “Nice enhancement, I'm eager to test it”
 
 - [SR/new] Clarify address formats [#4450](https://github.com/vatesfr/xen-orchestra/issues/4450) (PR [#4460](https://github.com/vatesfr/xen-orchestra/pull/4460))
+- [Menu] Remove the legacy backup entry [#4467](https://github.com/vatesfr/xen-orchestra/issues/4467) (PR [#4476](https://github.com/vatesfr/xen-orchestra/pull/4476))
```

Comment:
IMHO only one feature should be written in `CHANGELOG` after resolving all the issues related to #4467
So I don't think that this is relevant. @pdonias

Reply:
In theory, the Changelog should always be consistent with the code. So if something changes for the user, it should be mentioned, even if we plan to change a related piece of code in the next PR.

But then, in the next PR, the entries can be merged:
- After 1st PR: "Done this [Issue link] [PR1 link]"
- After 2nd PR: "Done this and that [Issue link] [PR1 link] [PR2 link]"